src/Fantomas.Core/CodePrinter.fs

```diff
@@ -1123,7 +1129,7 @@ let genExpr (e: Expr) =
                             short
                             (match leadingChain with
                              | [] -> sepNone
-                             | head :: links -> genFirstLinkAndIndentOther head links)
+                             | head :: links -> genLink false head +> indent +> genIndentedLinks true links +> unindent)
```

Comment:
@dawedawe I changed the implementation to only consider the `max_line_length` when there are only simple links in the chain.

You basically print the first one, if the second fits after it, great, if not put it on the next line. I think this is reasonable.

Reply:
Yep, very reasonable.